test/integration/integration_test.py

```diff
@@ -1,45 +1,13 @@
-#!/usr/bin/env python3
+#! /home/venv/bin/python
```

Comment:
Just for my edification: where is the virtual environment configured?

Reply:
Backed out the venv shebang: this is cruft from an experiment which I accidentally left in